src/dotnet/dotnet-fable/ProjectCracker.fs

```diff
@@ -47,15 +47,16 @@ type CrackedFsproj =
 
 let makeProjectOptions project sources otherOptions =
     { ProjectFileName = project
-      ProjectFileNames = sources
+      SourceFiles = sources
       OtherOptions = otherOptions
       ReferencedProjects = [| |]
       IsIncompleteTypeCheckEnvironment = false
       UseScriptResolutionRules = false
       LoadTime = DateTime.Now
       UnresolvedReferences = None
       OriginalLoadReferences = []
-      ExtraProjectInfo = None }
+      ExtraProjectInfo = None
+      Stamp = None }
```

Comment:
What's this? Does it have to do something with file caching when recompiling a project? I recently [tried setting LoadTime to Now for watch compilations](https://github.com/fable-compiler/Fable/commit/76c65fbc0c47a2937b21a72debd77c4fc0506bb3) to see if there's any performance improvement as recompilations may take quite long (around 10s) in some big projects, but I'm not sure it had any effect.